doc/proot/rpm-spec

```diff
@@ -10,7 +10,7 @@ Buildroot : %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 Prefix    : /usr
 Name      : proot
 
-BuildRequires: libtalloc-devel
+BuildRequires: libtalloc-devel, protobuf-c-devel, libattr-devel
```

Comment:
These should be declared as optional dependencies. Not sure off the top of my head how to do that with RPM.